pkg/agent/dispatcher.go

```diff
@@ -142,9 +147,14 @@ func (disp *Dispatcher) unregisterWaiter(id uint64) {
 
 // Make a request to the monitor and wait for a response. The value passed as message must be a
 // valid value to send to the monitor. See the docs for SerializeInformantMessage for more.
-func (disp *Dispatcher) Call(ctx context.Context, timeout time.Duration, message any) (*MonitorResult, error) {
+func (disp *Dispatcher) Call(ctx context.Context, timeout time.Duration, messageType string, message any) (*MonitorResult, error) {
```

Comment:
```suggestion
func (disp *Dispatcher) Call(
    ctx context.Context,
    timeout time.Duration,
    messageType string,
    message any
) (*MonitorResult, error) {
```
Line is getting a bit long.

Reply:
Good call. done 93bc7cf